metricq/history_client.py

```diff
@@ -42,17 +43,29 @@
 
 logger = get_logger(__name__)
 
-if TYPE_CHECKING:
```

Comment:
whoah, whoah, whoah, that stuff was added for mypi and typing or something. Does that still work fine?

Reply:
Yes, it does. Check out the commit message of 0c23658e2e3f237856a609065f574d967edd0bd8.

Reply:
Slight change in interface: values of `HistoryRequestType` are now `Enum`s instead of plain `int`s, but since the concrete type was never exposed, this should be fine.